src/test/java/no/nav/foreldrepenger/autotest/fpsak/foreldrepenger/SammenhengendeUttak.java

```diff
@@ -293,15 +294,30 @@ void utsettelse_med_avvik() {
         saksbehandler.hentFagsak(saksnummer);
         saksbehandler.ventTilAvsluttetBehandlingOgFagsakLøpendeEllerAvsluttet();
 
-        var startUtsettelse = fødselsdato.plusWeeks(6);
         var fordelingEndring = fordeling(
-                utsettelsesperiode(UtsettelsesÅrsak.ARBEID, startUtsettelse, startUtsettelse.plusWeeks(4).minusDays(1))
+                uttaksperiode(StønadskontoType.MØDREKVOTE, fødselsdato.plusWeeks(6), fødselsdato.plusWeeks(30).minusDays(1))
```

Comment:
Er dette en realistisk case?

Reply:
Nei, ikke direkte fra søknaden. Men er noen caser rundt fødselsjustering osv som man havner i den situasjonen.
Men det er enkleste måten å få et aksjonspunkt i uttak på nå som utsettelser tilbake i tid er ok

Reply:
Skjønner. Go for it 👌